src/feedback/FeedbackDialog.tsx

```diff
@@ -69,18 +70,33 @@ export const FeedbackDialog: FunctionComponent<FeedbackDialogProps> = ({
 
   let unanchoredPosition: Position = {};
   if (position.type === "DIALOG") {
+    const offsetLeft = parseOffset(position.offset?.left);
+    const offsetTop = parseOffset(position.offset?.top);
+
     switch (position.placement) {
       case "top-left":
-        unanchoredPosition = { top: "1rem", left: "1rem" };
+        unanchoredPosition = {
+          top: `calc(1rem + ${offsetTop})`,
+          left: `calc(1rem + ${offsetLeft})`,
```

Comment:
I'm not sure I like this. I think I would prefer to have the user defined offset be unmodified and us just having an `1rem` default that the user can replace

Reply:
Hmm, I was concerned it was pretty prone to human error. I revised the implementation to treat the offset as 'distance from the specified placement' rather than 'arbitrary CSS modifier' and that's simplified things